terraform-tests/README.md

```diff
@@ -1,23 +1,30 @@
-# Terraform Test Tools and Scripts
+# Test, Tools and Scripts
 
-Terraform tests are `unit` tests that run with the system's installed terraform binary on the terraform files directly. They will download the latest versions of the providers
-that comply with the restrictions in the provider.tf files. They won't look at the service offering definitions (.yml) or the manifest.yml files. 
+Tests within the `terraform-test` folder are `unit` tests that run with the system's installed OpenTofu binary on the
+OpenTofu language files directly. These tests download the latest versions of the providers that comply with the
+restrictions in the definition file. They do not examine the service offering definitions (.yml) or the manifest.yml
+files.
 
-The aim of these tests is to verify that given terraform gets a set of variables inputs, it will send the expected values to the providers. 
-This is accomplished by running `terraform plan --no-refresh` and analysing the output with terraform provided commands and structures. 
+The objective of these tests is to ensure that OpenTofu receives a set of variable inputs and sends the expected values
```

Comment:
Rather than `variables inputs` (bag grammar) or `variable inputs` (correct grammar, but means the inputs may vary) I think `input variables` is better because the inputs are variables.